### DISCUSSION TITLE
AI Art

### DISCUSSION THREAD

**Discussion by WEI-CHEN-CHUN** [1101-GS4719-C/Python-Community-GS4719-C#901] [General]
最近很流行的AI art，用戶上傳的照片，會有隱私權，版權的疑慮，儘管宣稱不會做其他用途。
另一個隱憂是，AI畫出來的版權是屬於AI還是公司又或者是上傳原圖者呢

文章連結：https://abc7news.com/lensa-ai-art-generator-image-app-prisma-labs/12540081/

**Comment by Kotori0501** [comment]
之前有試玩過NovelAi 的Ai繪圖的確版權爭議挺嚴重的，但如果像我一樣平時有在用電繪來自創角色，或許可以從AI繪圖中尋求一定的靈感

**Comment by peggy0329** [comment]
隨著AI技術越來越進步，各國對於隱私權相關的法規應該也要與時俱進，同時每家公司應該都要有專業的法遵人員來處理相關的問題。
隱私權問題真的是一個很有爭議的議題。

**Comment by ruthchen1020** [comment]
AI在藝術圈真的爭議頗多，畢竟有種二創人家作品的感覺，不問自拿當然會被詬病。

**Comment by StevenWanglolz** [comment]
AI藝術創作真的會帶來很多爭議，拿來訓練模型的藝術作品是否也需要經過創作者的同意呢?

**Comment by CPCpyt** [comment]
雖然AI的創作會帶來藝術的隱憂，
不過高度智能AI的藝術創作真的令人佩服，
也續在未來大家對「創作」的定義將會有巨大個改變。
